test/Sema/accessibility_shared_private.swift

```diff
@@ -0,0 +1,76 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: %utils/split_file.py -o %t %s
+// RUN: %target-swift-frontend -swift-version 4 -typecheck %t/declarations.swift %t/other_file_extensions.swift -verify
```

Comment:
([FileCheck docs](http://llvm.org/docs/CommandGuide/FileCheck.html))

I think using FileCheck instead of / in addition to -verify is probably the best way to go.I wouldn't worry about the notes, just the main errors.